test/functional/test_framework/mininode.py

```diff
@@ -48,8 +48,6 @@
 COIN = 100000000 # 1 btc in satoshis
 
 NODE_NETWORK = (1 << 0)
-NODE_GETUTXO = (1 << 1)
```

Comment:
Good point! Updated but GitHub doesn't show the `# NODE_GETUTXO = (1 << 1)` below, so it appears as if it is not fixed :-)